stdlib/public/runtime/CMakeLists.txt

```diff
@@ -99,7 +99,9 @@ if(SWIFT_BUILD_STATIC_STDLIB AND "${sdk}" STREQUAL "LINUX")
       "${SWIFT_SOURCE_DIR}/utils/${lowercase}/static-executable-args.lnk"
       "${SWIFTSTATICLIB_DIR}/${linkfile}"
     OUTPUT
-      "${SWIFTSTATICLIB_DIR}/${linkfile}")
+      "${SWIFTSTATICLIB_DIR}/${linkfile}"
+    DEPENDS
+      "${SWIFT_SOURCE_DIR}/utils/${lowercase}/static-executable-args.lnk")
```

Comment:
@spevans There is no ${lowercase} variable actually defined, so this always expands to the empty string.  That matches up with the source tree, where static-executable-args.lnk is just a file in utils/, but may not be what you wanted.